### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jockm** [darius/wren#1]
Is there any documentation for the Wren language, or other documentation on who to integrate the system into a larger system?

**Comment by darius** [comment]
No, I'm afraid there's just the comments in source code, plus the bits of examples.

I'm sorry for the late reply; I was on a trip where lots went wrong with connectivity, computers, etc. It's likely to be a long time before I get to this.

**Comment by jockm** [comment]
Thank you for getting back to me on this.  There is a lot about the syntax
that isn't clear at all.  Is white space relevant? What is up with the *
unary operator? The complete set of operators (I presume ult is unsigned
less than, but honestly there is no way to know), and so many more
questions.

I could carefully go through all the source and examples, and then
experiment; but I don't even have enough information to tell me if that is
worth it.  There just isn't enough there there; which was the answer in and
of itself.

Obviously there is only so much I can complain.  You were good enough to
put this out there, and for that I thank you; but a little documentation
would be nice.  If you ever find a free hour (and I doubt it would take
more than that), please go through and make a quick list of core syntax,
operators, and builtins.  That would go a long way.

All the best
Jock

On Sat, Jan 18, 2014 at 8:47 PM, Darius Bacon notifications@github.comwrote:

> No, I'm afraid there's just the comments in source code, plus the bits of
> examples.
> 
> I'm sorry for the late reply; I was on a trip where lots went wrong with
> connectivity, computers, etc. It's likely to be a long time before I get to
> this.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/darius/wren/issues/1#issuecomment-32701022
> .

## 

Jock Murphy
http://www.jockmurphy.com
Photographer \* Writer \* Storyteller
